jumpscale/core/base/meta.py

```diff
@@ -145,7 +145,7 @@ def setter(self, value):
         value = field.from_raw(value)
 
         # validate
-        field.validate(value)
+        field.validate_named(value, name)
```

Comment:
I think there're other `field.validate` references that needs  to be updated to this method too?